django_mongodb/client.py

```diff
@@ -4,11 +4,51 @@
 
 
 class DatabaseClient(BaseDatabaseClient):
-    executable_name = "mongo"
+    executable_name = "mongosh"
+    help_options = ("--help", "-h")
 
     @classmethod
     def settings_to_cmd_args_env(cls, settings_dict, parameters):
-        raise NotImplementedError
+        options = settings_dict.get("OPTIONS", {})
+        uri = options.get("uri")
+        args = [cls.executable_name]
+        if not uri:
+            host = settings_dict["HOST"]
+            port = settings_dict["PORT"]
+            if port and host:
+                host += f":{port}"
+
+            dbname = settings_dict["NAME"]
+            user = settings_dict["USER"]
+            passwd = settings_dict["PASSWORD"]
+            auth_database = options.get("authenticationDatabase")
+            auth_mechanism = options.get("authenticationMechanism")
+            retry_writes = options.get("retryWrites")
+            protocol = options.get("protocol", "mongodb")
```

Comment:
Again, there is no "protocol" kwarg to MongoClient.

Reply:
Do we need here to support mongosh or pymongo's mongoclient?
I think We have to use mongosh which is the equivalent of psql in Postgres.

Reply:
Also, a protocol could be under the host URL. So, it isn't a parameter itself but is contained in host parameter.